retrofit-mock/src/main/java/retrofit2/mock/NetworkBehavior.java

```diff
@@ -46,12 +46,12 @@ public final class NetworkBehavior {
   private static final int DEFAULT_VARIANCE_PERCENT = 40; // Network delay varies by ±40%.
   private static final int DEFAULT_FAILURE_PERCENT = 3; // 3% of network calls will fail.
   private static final int DEFAULT_ERROR_PERCENT = 0; // 0% of network calls will return errors.
-
+  
```

Comment:
Here and elsewhere, try to avoid adding unnecessary diffs like this whitespace change.